thefuck/rules/git_not_command.py

```diff
@@ -14,5 +22,9 @@ def match(command):
 def get_new_command(command):
     broken_cmd = re.findall(r"git: '([^']*)' is not a git command",
                             command.output)[0]
-    matched = get_all_matched_commands(command.output, ['The most similar command', 'Did you mean'])
+    # check if the broken_cmd exists in COMMON_TYPOS
+    if broken_cmd in COMMON_TYPOS:
+        matched = COMMON_TYPOS[broken_cmd]
+    else:
+        matched = get_all_matched_commands(command.output, ['The most similar command', 'Did you mean'])
```

Comment:
We still want to suggest similar commands. Hence the call to `extend` in my [suggestion](https://github.com/nvbn/thefuck/pull/883#pullrequestreview-201147994).